### PR TITLE
TATB - 37 : Redesign Quiz Test Template

### PR DESCRIPTION
**Pull request by PrabhatJTDeveloper**

### CODE REVIEW

File: src/app/component/quiz/quiz.component.html [hunk 1]
```diff
@@ -24,121 +21,132 @@
...
-                        aria-hidden="true"
-                        data-bs-slide="per"
-                        style="
-                          position: absolute;
-                          left: 40%;
+                          <div class="form_btn">
+                            <button
+                              [disabled]="i == 0"
+                              class="carousel-control-Previous"
+                              id="previous"
```
Comment (charmijtdev) [inline src/app/component/quiz/quiz.component.html:106]: what is use for this id and role and all ?

Comment (charmijtdev) [reply src/app/component/quiz/quiz.component.html:106]: data-slide="Previous"
 aria-hidden="true"
 data-bs-slide="per"

Comment (charmijtdev) [reply src/app/component/quiz/quiz.component.html:106]: Please check in all and remove if not required

File: src/app/component/quiz/quiz.component.scss [hunk 2]
```diff
@@ -0,0 +1,130 @@
+* {
+  font-family: Roboto;
+}
+.quiz_card {
+  height: auto;
+  // max-height: calc(100vh - 150px);
```
Comment (charmijtdev) [inline src/app/component/quiz/quiz.component.scss:6]: remove unwanted css

File: src/app/component/quiz/quiz.component.scss [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+
+.option {
+  background: rgb(222, 239, 255);
+  border: 1px solid #84c5fe;
+  border-radius: 5px;
+  padding: 8px 15px;
+  font-size: 17px;
+  cursor: pointer;
+  margin-top: 15px;
+  // margin-bottom: 15px;
```
Comment (charmijtdev) [inline src/app/component/quiz/quiz.component.scss:79]: please remove unwanted css

File: src/app/component/quiz/quiz.component.scss [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+}
+
+.total_que p {
+  text-align: center;
+  margin: 0;
+  user-select: none;
+  padding: 0 5px;
+}
+
+.total_que span p:first-child {
```
Comment (charmijtdev) [inline src/app/component/quiz/quiz.component.scss:128]: please do not use like span p:

**Review by charmijtdev — CHANGES_REQUESTED** [review]

File: src/app/component/quiz/quiz.component.scss [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+  margin-top: 15px;
+  // margin-bottom: 15px;
+  align-items: center;
+  width: 100%;
+}
+.hightlight-selected {
+  background: rgb(133, 190, 255) !important;
+  border: 1px solid #4a98ff !important;
+}
+input[type="radio"],
```
Comment (charmijtdev) [inline src/app/component/quiz/quiz.component.scss:87]: do not use directly like this instead use boostrap class d-none or use

**Review by charmijtdev — CHANGES_REQUESTED** [review]

File: src/app/component/quiz/quiz.component.html (hunk 1 above)
Comment (PrabhatJTDeveloper) [reply src/app/component/quiz/quiz.component.html:106]: yes, i removed which are not required.

File: src/app/component/quiz/quiz.component.scss (hunk 2 above)
Comment (PrabhatJTDeveloper) [reply src/app/component/quiz/quiz.component.scss:6]: ok, i removed it.

File: src/app/component/quiz/quiz.component.scss (hunk 3 above)
Comment (PrabhatJTDeveloper) [reply src/app/component/quiz/quiz.component.scss:79]: ok, i removed it.

File: src/app/component/quiz/quiz.component.scss (hunk 4 above)
Comment (PrabhatJTDeveloper) [reply src/app/component/quiz/quiz.component.scss:128]: ok, i removed it.

File: src/app/component/quiz/quiz.component.scss (hunk 5 above)
Comment (PrabhatJTDeveloper) [reply src/app/component/quiz/quiz.component.scss:87]: ok, i removed it from scss and gave it the boostrap class

**Comment by charmijtdev** [comment]
OLD PR

**Pull request closed without merging**